drivers/project_scorecard/app/controllers/project_scorecard/warehouse_reports/scorecards_controller.rb

```diff
@@ -215,6 +233,36 @@ def project_group_scope
       @filter = ::Filters::FilterBase.new(initial_filter_params.merge(user_id: current_user.id, project_type_codes: []))
     end
 
+    private def filtered_reports_scope
+      scope = reports_scope.started_between(start_date: @history_filter.start, end_date: @history_filter.end)
+
+      scope = scope.where(project_id: project_scope.select(:id)).
+        or(scope.where(project_group_id: project_group_scope.select(:id)))
```

Comment:
These are filtering for viewability. We only really need to do it if `@history_filter.project_ids` and `@history_filter.project_group_ids` are both empty, I think? (Although even if they are filled, is it possible they could include a report that is not viewable, by messing with query params? )